cmd/crank/xpkg/install.go

```diff
@@ -50,27 +50,44 @@ const (
 	errKubeClient    = "failed to create kube client"
 )
 
-// InstallCmd is exported so that it can be re-used by the beta xpkg subcommand.
+// installCmd installs a package.
+type installCmd struct {
+	// Arguments.
+	Kind  string `arg:"" help:"The kind of package to install. One of \"provider\", \"configuration\", or \"function\"." enum:"provider,configuration,function"`
+	Image string `arg:"" help:"The OCI image name of the package to install."`
+	Name  string `arg:""  optional:"" help:"The name of the new package resource in the Crossplane API. Derived from the package's OCI image name by default."`
+
+	// Flags. Keep sorted alphabetically.
+	RuntimeConfigName    string        `placeholder:"NAME" help:"The name of the runtime config the package should use."`
+	ManualActivation     bool          `short:"m" help:"Require the new package's first revision to be manually activated."`
+	PackagePullSecrets   []string      `placeholder:"NAME" help:"A comma-separated list of secret names the package manager should use to pull the package from the registry."`
+	RevisionHistoryLimit int64         `short:"r" placeholder:"LIMIT" help:"How many package revisions may exist before the oldest revisions are deleted."`
+	Wait                 time.Duration `short:"w" default:"0s" help:"How long to wait for the package to install before returning. The command does not wait by default."`
```

Comment:
```
      --package-pull-secrets=NAME,...
                               A comma-separated list of secrets the package
                               manager should use to pull the package from the
                               registry.
```

I tweaked this one but want to leave the rest as is. I feel they mostly better describe what the flags actually do. I considered setting `default:"1"` (or 2 actually) but this would actually set that limit every time. I'd prefer to defer to the Crossplane API's existing defaults (which I think is 2).